conf/defaults.ini

```
@@ -369,6 +369,9 @@ content_security_policy_report_only_template = """script-src 'self' 'unsafe-eval
 # Controls if old angular plugins are supported or not. This will be disabled by default in future release
 angular_support_enabled = true
 
+# The CSRF check will be executed even if the request has no login cookie.
+csrf_always_check = true
```

```suggestion
csrf_always_check = false
```

align with @papagian's comment

---

Fixed